worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/listener/EventAbstractionListener.java

```diff
@@ -1161,7 +1161,7 @@ public void onCauldronLevelChange(CauldronLevelChangeEvent event) {
         }
 
         // handle water/lava placement
-        if (item != null && (item.getType() == Material.WATER_BUCKET || item.getType() == Material.LAVA_BUCKET)) {
+        if (item != null && (item.getType() == Material.WATER_BUCKET || item.getType() == Material.LAVA_BUCKET || Materials.isShulkerBox(item.getType()))) {
             Events.fireToCancel(event, new PlaceBlockEvent(event, cause, placed.getLocation(),
                     item.getType() == Material.WATER_BUCKET ? Material.WATER : Material.LAVA));
```

Comment:
shulker isn't handled here, which is fine for our region protection but breaks stuff like blacklist :< (and potential third party handling, not that we're super-good about that anyway)

Reply:
Uh, yeah i forgot to add that here.

Reply:
I have added a check for dispensers, which can also be used for other special dispenser behavior in the future.